aip-console-tools-cli/src/main/java/com/castsoftware/aip/console/tools/commands/AddVersionCommand.java

```diff
@@ -136,6 +136,7 @@ public AddVersionCommand(RestApiService restApiService, JobsService jobsService,
     @Override
     public Integer call() {
         ApiInfoDto apiInfo = null;
+        restApiService.setVerbose(sharedOptions.isVerbose());
```

Comment:
should not do that in each class

Reply:
You're right. I'll add it in a base case class like we already do for Jenkins plugin